packages/kolibri-common/components/SyncSchedule/EditDeviceSyncSchedule.vue

```diff
@@ -71,7 +71,7 @@
 
           <p class="spacing">
             <KCheckbox>
-              {{ $tr('checboxlabel') }}
+              {{ $tr('checkboxlabel') }}
```

Comment:
I flagged elsewhere that this should probably be `'checkboxLabel'` (both here and where it's defined).